sdk/client.go

```diff
@@ -59,22 +60,23 @@ func (c *Client) postResource(resPath string, requestBody interface{}) ([]byte,
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusCreated {
-		return nil, fmt.Errorf("unexpected status: %d", resp.StatusCode)
-	}
-
 	body, err := io.ReadAll(resp.Body)
 	if err != nil {
 		return nil, err
 	}
 
-	return body, nil
+	var statusErr error
+	if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusCreated {
+		statusErr = fmt.Errorf("unexpected status: %d", resp.StatusCode)
+	}
+
+	return body, statusErr
 }
 
 func (c *Client) getServiceStatus(resPath string) (*api.MetaMessage, error) {
-	resp, err := c.getResource(resPath)
-	if err != nil {
-		return nil, err
+	resp, resErr := c.getResource(resPath)
```

Comment:
i would definitely keep to just calling these `err` all the time as its easier to just scan them and ignore as "error handling boilerplate"

Reply:
good catch, pushed a fix